tools/split_suggester.py

```diff
@@ -513,17 +609,33 @@ def userConfirms():
 
 
 # Print results
-print("Total functions: " + str(len(functions)))
-print("Total floats:    " + str(len(floatTypes)))
-print("Text Section:    0x{:X} bytes".format(textBytes))
-print("Data Section:    0x{:X} bytes".format(dataBytes))
-print("sdata2 Section:  0x{:X} bytes".format(sdataBytes))
+print("  Total functions:            " + str(len(functions)))
```

Comment:
Please move the main execution stuff to an `if __name__ == "__main__"` block.

Reply:
Done.